.github/workflows/build.yml

```diff
@@ -2,6 +2,8 @@ name: Build
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+    paths-ignore:
+      - 'infra/**'
```

Comment:
I'd like all our deployed resources to be under one root dir, does it make sense to have a structure like:
````
infrastructure/
├── lambda/
├── parameter/
````
We can defer this to another clean-up PR though if it muddies this PR.

Reply:
Infrastructure/lambda, would this be both the sam template and the api yaml?